### ISSUE TITLE
File Manager field does not load with fieldsForUpdate

### ISSUE THREAD

**Issue by iMuMz** [oneduo/nova-file-manager#208]
**Describe the bug**
The file manager field does not seem to load when the field is placed inside the fieldsForUpdate method. The normal fields method works fine.

**To Reproduce**
Add the file manager field to the fieldsForUpdate method.

```
public function fieldsForUpdate(NovaRequest $request) {
  return [
     FileManager::make('Document')
  ];
}
```

**Expected behavior**
I expect the file manager to load as normal.

**Screenshots**
![Screenshot 2023-03-23 at 08 18 00](https://user-images.githubusercontent.com/22936672/227119991-37137bf7-f6a4-47c9-94e2-bcf202881d43.png)

**Desktop (please complete the following information):**

- OS: MacOs
- Browser Chrome
- Version Version 111.0.5563.64 (Official Build) (x86_64)

**Comment by Rezrazi** [comment]
HI

This should be fixed in [0.9.1](https://github.com/oneduo/nova-file-manager/releases/tag/v0.9.1)

Please do let us know if it's working as intended now

Cheers

**Comment by iMuMz** [comment]
@Rezrazi It now works as intended. Thank you, I appreciate it. Really great package
